permissions/plugin-jackson2-api.yml

```diff
@@ -8,6 +8,7 @@ issues:
 paths:
 - "org/jenkins-ci/plugins/jackson2-api"
 developers:
+- "@cloudbees-developers"
```

Comment:
this plugin is using CD so there should be no need to grant this access - infact user level access should probably be revoked?

possibly something to do as a follow up for all plugins?

Reply:
up to plugins, generally people leave it in there in case there's issues and they need to release manually

Reply:
That is what we have done historically, especially when JEP-229 was brand new and we were still unsure whether it would work reliably. Usage on backports still remains to be validated AFAIK (https://github.com/jenkins-infra/jenkins-maven-cd-action/issues/4). Then on the other hand there are security arguments for revoking manual access to components with `cd.enabled`. A broader discussion that could be taken to the list I guess.